lms/static/js/learner_dashboard/models/course_card_model.js

```diff
@@ -115,12 +115,30 @@
                     return DateUtils.localize(context);
                 },
 
+                getCertificatePriceString: function(run) {
+                    var verified_seat, currency;
+                    if ('seats' in run && run['seats'].length) {
+                        verified_seat = _.filter(run['seats'], function(seat){
+                            if (seat['type'] === 'verified' || seat['type'] === 'professional' || seat['type'] === 'credit'){
+                                return seat;
+                            }
+                        })[0]
+                        currency = verified_seat['currency'];
+                        if (currency === 'USD'){
+                            return '$' + verified_seat['price'];
+                        } else {
+                            return verified_seat['price'] + " " + currency;
```

Comment:
Nitpick: single quotes should be used instead of double wherever possible in keeping with the coding style of the repo.